src/models/GOKU.jl

```diff
@@ -5,8 +5,21 @@
 
 struct GOKU <: LatentDE end
 
+@doc raw"""
+    apply_feature_extractor(encoder, x)
+Converts a batch of the initial high-dimensional data into lower-dimensional data (i.e. extracts features)
```

Comment:
Although usually this step converts  high-dimensional data into lower-dimensional data, this may not always be the case. The feature_extractor layer could be set up for doing some transformation but keeping the dimensionality. I know it's not the original idea, but the possibility exists. So I would say something like: 

Applies [or some other verb] the feature extractor layer contained in the `encoder` to the batch of input data `x`, usually reducing its dimensionality (i.e. extracting features).

Reply:
Yes, that's nice.  I will make that change.